Client/Ecosia/UI/MyImpact/MultiplyImpactCell.swift

```diff
@@ -5,82 +5,87 @@
 import UIKit
 
 final class MultiplyImpactCell: UICollectionViewCell, AutoSizingCell, Themeable {
-    override init(frame: CGRect) {
-        super.init(frame: frame)
-        setup()
-    }
+    private(set) weak var widthConstraint: NSLayoutConstraint!
+    private weak var title: UILabel!
+    private weak var subtitle: UILabel!
+    private weak var outline: UIView!
 
-    required init?(coder: NSCoder) {
-        super.init(coder: coder)
+    override var isSelected: Bool {
+        didSet {
+            hover()
+        }
     }
 
-    private weak var stack: MyImpactStackView!
-    private weak var outline: UIView!
-    private var widthConstraint: NSLayoutConstraint!
-
-    private func setup() {
+    override var isHighlighted: Bool {
+        didSet {
+            hover()
+        }
+    }
+    
+    required init?(coder: NSCoder) { super.init(coder: coder) }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
         let outline = UIView()
         contentView.addSubview(outline)
         outline.layer.cornerRadius = 8
         outline.translatesAutoresizingMaskIntoConstraints = false
         self.outline = outline
+        
+        let title = UILabel()
+        title.translatesAutoresizingMaskIntoConstraints = false
+        title.font = .preferredFont(forTextStyle: .body)
+        title.adjustsFontForContentSizeCategory = true
+        title.numberOfLines = 0
+        title.text = .localized(.getATreeWithEveryFriend)
+        self.title = title
+        outline.addSubview(title)
+        
+        let subtitle = UILabel()
+        subtitle.translatesAutoresizingMaskIntoConstraints = false
+        subtitle.font = .preferredFont(forTextStyle: .callout)
+        subtitle.adjustsFontForContentSizeCategory = true
+        subtitle.text = .localized(.inviteFriends)
+        self.subtitle = subtitle
+        outline.addSubview(subtitle)
+        
+        let icon = UIImageView(image: .init(themed: "groupYourImpact"))
```

Comment:
if it's a themed image, you would want to update the image in `applyTheme()` as well. Otherwise it might miss dynamic theme changes